lib/benchpark/accounting.py

```diff
@@ -7,36 +7,52 @@
 
 import benchpark.paths
 
+exclude_exper = ["repo.yaml"]
+
 
 def benchpark_experiments():
     source_dir = benchpark.paths.benchpark_root
     experiments = []
-    experiments_dir = source_dir / "legacy" / "experiments"
-    for x in os.listdir(experiments_dir):
-        for y in os.listdir(experiments_dir / x):
-            experiments.append(f"{x}/{y}")
+    experiments_dir = source_dir / "experiments"
+    exclude_variants = ["Caliper"]
+    for x in sorted(os.listdir(experiments_dir)):
+        if x not in exclude_exper:
+            experiment_spec = benchpark.spec.ExperimentSpec(x)
+            conc = experiment_spec.concretize()
+            experiment_class = conc.experiment
+            for h in experiment_class.__dict__["helpers"]:
+                variant = str(h).split(".")[2]
+                if variant not in exclude_variants:
+                    experiments.append(f"{x}/{variant}")
     return experiments
 
 
 def benchpark_modifiers():
     source_dir = benchpark.paths.benchpark_root
     modifiers = []
-    for x in os.listdir(source_dir / "modifiers"):
-        modifiers.append(x)
-
-    modifiers += [
-        x for x in os.listdir(source_dir / "legacy" / "modifiers") if x not in modifiers
-    ]
+    exclude = ["modifier_repo.yaml"]
+    for x in sorted(os.listdir(source_dir / "modifiers")):
+        if x not in exclude:
+            modifiers.append(x)
 
     return modifiers
 
 
 def benchpark_systems():
     source_dir = benchpark.paths.benchpark_root
     systems = []
-    for x in os.listdir(source_dir / "legacy" / "systems"):
-        if not (
-            os.path.isfile(os.path.join(source_dir / "configs", x)) or x == "common"
-        ):
```

Comment:
I think it makes more sense now for `benchpark list systems` to be 
```
Systems:
        aws-pcluster
        csc-lumi
        cscs-daint
        cscs-eiger
        ...
```
Instead of like the legacy way
```
Systems:
        AWS_PCluster-zen-EFA
        Atos-rome-A100-Infiniband
        CSC-LUMI-HPECray-zen3-MI250X-Slingshot
        CSCS-Daint-HPECray-haswell-P100-Infiniband
        CSCS-Eiger-HPECray-zen2-Slingshot
        ...
```
since `benchpark system init` works using the former, e.g.
`benchpark system init --dest=ruby-system llnl-cluster`

Reply:
Try
```
Systems:
    llnl-cluster:
         - ruby
         - dane
         - ...
```